app/api/contact/contact.js

```diff
@@ -5,7 +5,7 @@ export default {
   async sendMessage({ email, name, message }) {
     const siteSettings = await settings.get();
     const mailOptions = {
-      from: '"Uwazi" <no-reply@uwazi.io',
+      from: '"Uwazi" <no-reply@uwazi.io>',
```

Comment:
Not sure if this is the time to address this, but the fact that we send all our mails as "Uwazi" is kind of confusing. People use a document management platform "powered by Uwazi", but a lot of users would expect the mail to come from "Plan International", or "CEJIL", not from "Uwazi", as 90% of the people using the platform (other than admins) have no idea what Uwazi is.

Could we just change this to the collection name? But some collection names are whole books, so not sure. Perhaps a configurable option (another?) in collection settings? Should we do this here and now? (especially since this opens up the scenario to sending of a LOT more emails).

Reply:
I agree with the problem, but that hardcoded email is for the contact form, I was just adding a missing ">"

The public form will use a "From" coming from the custom JS for the custom public form.
Adding the option to setings will be nice, but since that is the only form using that functionality, I think that adding that option in settings should be done when we modify the PublicForm to be able to configure the send an email option, or we will be adding an. option to settings that no one can use except for the contact form but I think if we do it for the contact form, it should be in another issue to do not extend this one.

Reply:
Yes, I agree.  That's why I mentioned this was probably not the place for this discussion.

Reply:
Created issue #2912 to track this development.